### ISSUE TITLE
Page jumping around with url fragments

### ISSUE THREAD

**Issue by davidrapson** [guardian/frontend#10790]
1: Append a `#` to any page, e.g. http://www.theguardian.com/uk#
2: Reload the page
3: Watch the page have a scroll party

Demo:

![header-freakout mov](https://cloud.githubusercontent.com/assets/123386/10315683/aa647aca-6c53-11e5-8fa2-570035dc3c43.gif)

This only happens with a single `#` at the end of the url, so an educated guess would be that this is a side-effect of some hash/anchor listening code which gets confused when there is no anchor. Maybe.

**Comment by stephanfowler** [comment]
Yikes. Seems it could be related to ads? Because:
1. The scroll seems to jump to the top of an ad unit.
2. It happens when in "core" too (which basically loads ads and little else).
@uplne @Calanthe

**Comment by ScottPainterGNM** [comment]
Flagged for commercial - if this also happens in core it may not be around the sticky-nav but flag can be re-added if needed.

**Comment by ScottPainterGNM** [comment]
This is still very much present, will ask for it to be added to the commercial health queue

**Comment by JonNorman** [comment]
I've looked into this a bit but alas without success:

1. this is almost certainly ad related as @stephanfowler mentioned - the screen jumps to the top of each ad and this behaviour does not occur when using an ad-blocker.
2. this isn't unique to our site and happens on a bunch of other sites that I checked such as http://www.theatlantic.com/world/# and http://www.dailymail.co.uk#

I looked for empty blank targeting elements - i.e. `href="#"` and other similar entities - within the delivered ads but couldn't find them.

I'm out of ~~bright~~ ideas.
